e2e/colorpicker.e2e-spec.ts

```diff
@@ -62,6 +62,12 @@ describe('Colorpicker', () => {
     });
   }
 
+  async function validateColorpickerDisabled(done: DoneFn): Promise<void> {
+    expect('#screenshot-colorpicker3').toMatchBaselineScreenshot(done, {
+      screenshotName: getScreenshotName('colorpicker-disabled')
+    });
+  }
+
```

Comment:
the helper function that validates that the disabled colorpicker looks as it should